app/models/customer_item.rb

```diff
@@ -0,0 +1,11 @@
+class CustomerItem < ApplicationRecord
+  belongs_to :customer
+  belongs_to :item
+
+  def add_item
+    customer = Customer.find(params[:id])
```

Comment:
You can definitely encapsulate this logic into a method in the Model, however, the model does not have access to the params hash, that's only in the controller. So to make this work, you'd need to have this method accept two arguments `def add_item(customer_id, item_id)` and then when you're calling this method from the Controller, it would look like this: `CustomerItem.add_item(params[:id], params[:item_id])`